build.assets/tooling/cmd/notarize-apple-binaries/hashicorpLoggerAdapter.go

```diff
@@ -0,0 +1,228 @@
+/*
```

Comment:
YMMV - is this worth breaking out into `tooling/lib`, as this is plumbing that isn't _directly_ related to signing the Darwin builds? Just to reduce clutter in `notarize-apple-binaries`.

Reply:
Good idea - updated